integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/altinn/MessagePolling.java

```diff
@@ -113,7 +111,7 @@ public void checkForFiksMessages() {
     @Scheduled(fixedRate = 15000)
     public void checkForNewMessages() throws MessageException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "checkForNewMessages" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)